tests/test_other.py

```diff
@@ -666,6 +666,19 @@ def test_cmake_compile_features(self):
     emscripten_features = '\n'.join([x for x in emscripten_features.split('\n') if '***' in x])
     self.assertTextDataIdentical(native_features, emscripten_features)
 
+  # Test that concat generator arguments(e.g. "-GNinja" instead of "-G Ninja") can be correctly recognized passed to cmake
+  # Windows Only
+  def test_cmake_with_concat_generator_arg(self):
```

Comment:
How about just calling this "test_cmake_explicit_generator".

And for the comment how about:

"Test that the users explicitly specified generator is always honored.  Internally we override the generator on windows, but we should not do this when the user specifies one."

Reply:
Gotcha, then I gonna change to that one. I've been racking my brain trying to give such a special test a nice name ; )